source/fundamentals/indexes.txt

```diff
@@ -0,0 +1,424 @@
+
+Wildcard Indexes
+~~~~~~~~~~~~~~~~
+
+**Wildcard indexes** enable queries against unknown or arbitrary fields.
+These indexes can be beneficial if you are using a dynamic schema.
+
+The following example creates an ascending wildcard index on all
+values of the ``location`` field within the ``sample_mflix.theaters`` collection,
+including values nested in subdocuments and arrays:
+
+.. literalinclude:: /includes/fundamentals/code-examples/Indexes.cs
+   :language: csharp
+   :start-after: begin-wildcard-index
+   :end-before: end-wildcard-index
+   :dedent:
+
+For more information, see the :manual:`Wildcard Indexes</core/index-wildcard>` page in the
+Server manual.
+
+.. _search-indexes:
+
+Atlas Search Indexes
+~~~~~~~~~~~~~~~~~~~~
+
+The Atlas Search feature enables you to perform full-text searches on collections hosted
+on MongoDB Atlas. The indexes specify the behavior of the search and which fields to
+index.
+
+To learn more about MongoDB Atlas Search, see the :atlas:`Atlas Search Indexes </atlas-search/atlas-search-overview/#fts-indexes>`
+documentation.
+
+.. note::
+
+   The Atlas Search Index management methods run asynchronously. The
+   driver methods can return before confirming that they ran
+   successfully. To determine the current status of the indexes, call the
+   `IMongoSearchIndexManager.List() <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.Search.IMongoSearchIndexManager.List.html>`__ method.
+
+The following sections contain links to tutorials that demonstrate how to create and
+interact with Atlas Search indexes.
+
+Create a Search Index
++++++++++++++++++++++
+
+Before you can perform a search on an Atlas collection, you must first
+create an Atlas Search index on the collection. To learn how to create an Atlas Search
+index using the {+driver-short+}, see :atlas:`Create an Atlas Search Index </atlas-search/create-index/>`
+in the Atlas manual and select :guilabel:`C#` from the language dropdown.
+
+List Search Indexes
++++++++++++++++++++
+
+To learn how to view a list of your Atlas Search indexes using the {+driver-short+}, see
+:atlas:`View an Atlas Search Index </atlas-search/view-index/>` in the Atlas manual
+and select :guilabel:`C#` from the language dropdown.
+
+Update a Search Index
++++++++++++++++++++++
+
+To learn how to modify an existing Atlas Search index using the {+driver-short+}, see
+:atlas:`Edit an Atlas Search Index </atlas-search/edit-index/>` in the Atlas manual
+and select :guilabel:`C#` from the language dropdown.
+
+Drop a Search Index
++++++++++++++++++++
+
+To learn how to delete an Atlas Search index using the {+driver-short+}, see
+:atlas:`Delete an Atlas Search Index </atlas-search/delete-index/>` in the Atlas manual
+and select :guilabel:`C#` from the language dropdown.
+
+List Indexes
+------------
+
+You can use the `IMongoIndexManager<TDocument>.List() <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.IMongoIndexManager-1.List.html>`__
```

Comment:
I'm not sure we need to list the interface here, imo it just makes things less clear since you don't actually call it like that in the code.

```suggestion
You can use the `List() <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.IMongoIndexManager-1.List.html>`__
```